### ISSUE TITLE
WEMOS forum not working

### ISSUE THREAD

**Issue by mike2nl** [wemos/D1_mini_Examples#44]
@wemos 

Hey guys from Wemos,
You wrote on the main page about the Forum. http://forum.wemos.cc/

- That forum is not working now for more then 6 month.
- What's going on with you guys?

**Comment by jonhp** [comment]
Yes, I just tried this on Sept 5, 2024 and it is still a dead link. Not very encouraging.

**Comment by Mergenth** [comment]
agree. You could check Carenuity. They have various sample codes and are supportive.
